src/ResourceManager/DataLakeStore/Commands.DataLakeStore/DataPlaneModels/DataLakeStoreFileSystemClient.cs

```diff
@@ -45,8 +45,17 @@ public class DataLakeStoreFileSystemClient
         private const int MaxConnectionLimit = 1000;
         private const long NeverExpireValue = 253402300800000;
         internal const int ImportExportMaxThreads = 256;
+        private readonly LoggingConfiguration _adlsLoggerConfig;
+
         #region Constructors
 
+        static DataLakeStoreFileSystemClient()
+        {
+            // ConfigurationItemFactory.Default.Targets.RegisterDefinition("AdlsLogger", typeof(AdlsLoggerTarget));
```

Comment:
@rahuldutta90 nit: would you mind removing this line if it's going to be commented-out?

Reply:
@cormacpayne  Removed. Thanks for catching it.